src/components/Nav/Nav.jsx

```diff
@@ -22,7 +22,7 @@ const Nav = ({
 }) => {
   const [hasBorder, setHasBorder] = useState(false)
   const handleScroll = () => {
-    setHasBorder(window.pageYOffset > 15)
+    setHasBorder(window.pageYOffset > 15 || window.innerWidth <= 768)
```

Comment:
fixed this in #6, was a css issue were margin-top was being applied to the hero, causing the navbars transparent background to be located over white blackspace. Can remove js bug fix as this is css issue

Reply:
Ok sounds good.